main_test.go

```diff
@@ -451,10 +451,112 @@ func TestPublishersEndpoints(t *testing.T) {
 				// TODO: check there are no dangling publishers_codeHosting
 			},
 		},
+		{
+			query: "POST /v1/publishers - with externalCode example",
```

Comment:
```suggestion
			description: "POST publishers - with externalCode example",
			query: "POST /v1/publishers",
```

Let's always use `description` in tests (also the ones below).
Using `query` like that works by pure chance.